assets/styles/layouts/_profile.scss

```diff
@@ -20,11 +20,31 @@
   display: none;
 }
 
-#item-body > div.donut > svg {
-  min-width: 600px;
-  min-height: 300px;
-  width: 100%;
+#item-body div.donut > svg{
+  min-height: 200px;
+  min-width: 200px;
+  max-width: 300px;
 }
+
+#item-body div.donut-legend {
+  .legend-item{
```

Comment:
- pedantic formatting/spacing issues can be addressed (depending on your IDE) via keyboard shortcut https://www.jetbrains.com/help/phpstorm/using-code-editor.html#reformat_rearrange_code

Reply:
ugh spaces everywhere! Thanks for the tip.